dist/files/ovn-kubernetes.sysconfig

```diff
@@ -0,0 +1,27 @@
+# ovn-kubernetes config - all nodes and masters
+
+# This file configures the ovn cni plugin. When completed or changed
+# the file must be copied to each ovn-master and each ovn-node. The
+# ovn-kubernetes-master and ovn-kubernetes-node daemons must be restarted.
+
+# This file and /etc/openvswitch/ovn_k8s.conf, in combination, provide
+# all of the configuration options. See man 5 ovn_k8s.conf
+
+# The token can be created from information on the Openshift master
+# after Openshift is installed and the master is running.
+# Login to the openshift master and do the following commands once 
+# to set up openshift for ovn. The token can be set either here or in 
+# ovn_k8s.conf
+
+# one time setup
```

Comment:
We already have documented the way to create a token. So we do not need it here maybe? If we do keep it here, we must specify the kubectl way of setting it up also.

Reply:
Deleted the notes on how to prepare openshift to use this.
It is, after all, intuitively obvious to the most casual observer.